pom.xml

```diff
@@ -64,9 +64,14 @@
     <maven.version>3.8.8</maven.version>
 
     <flink.version>1.14.6</flink.version>
-    <hadoop.version>3.2.4</hadoop.version>
     <spark.version>3.3.2</spark.version>
 
+    <!-- use hadoop-3 as default  -->
+    <hadoop.version>3.2.4</hadoop.version>
+    <hadoop-client-api.artifact>hadoop-client-api</hadoop-client-api.artifact>
```

Comment:
I suppose the celeborn-mr-client-shaded built a against hadoop3 is runnable on hadoop2 too, thus, can we eliminate the global change and create a dedicated module under `tests/mr-it-hadoop2`, which launches a hadoop2 cluster via hadoop minicluster and tests by submitting an MR job with celeborn-mr-client-shaded built against hadoop3.

Reply:
Won't do this. Running mr integration test will need to shade celeborn worker and master, this is awful.